### ISSUE TITLE
Is Live Reload supported?

### ISSUE THREAD

**Issue by stevendevit** [chromelyapps/Chromely#197]
Hi. I want to reload a page when the html file is edited. I tried ReloadIgnoreCache() but is not working, it keeps loading the same content. 

Any idea? Thanks

**Comment by mattkol** [comment]
@stevendevit

So I assume you are using it as in https://github.com/chromelyapps/Chromely/issues/140

Not a feature, I have used personally - but I found this - https://bitbucket.org/chromiumembedded/cef/issues/1420#comment-23792535.
Down in the page it has:

> Workaround : Execute this JavaScript code to reload that page location.reload();

That may be helpful.

The other trick I remember from my Silverlight days is to add different fake query parameter at the end of the url for a fresh reload.
